_posts/2024-03-25-webkigit-2.44.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: What's new in WebKitGTK 2.44?
+---
+
+Last week the WebKitGTK team released version 2.44 of the GTK port of the WebKit project. This release is an important milestone for the project as it's the first where GTK4 is the default toolkit. Let's have a look to what's new in this new release.
+
+### GTK4 is now used by default
+
+While [the GTK4 API has been stable for about a year now](https://blogs.gnome.org/mcatanzaro/2023/03/21/webkitgtk-api-for-gtk-4-is-now-stable/), and the development releases leading up to 2.44 have already been building with GTK4 support by default, this is the first stable release that features GTK4 prominently. The GTK3 API is still available, and it's possible to parallel-install WebKitGTK targetting both libraries, but we encourage everyone to start migrating to the new API, testing it, and to [report any issues you might find](https://bugzilla.webkit.org).
+
+### Improved accessibility support with GTK4
+
+One of the main missing pieces for accessibility in the GTK4 port was being able to connect the accessibility tree for the web content to the rest of the hierarchy exposed by GTK4 to accessibility technologies. During this cycle [Georges Stavracas](https://feaneron.com) coordinated with the GTK developers to add new API that WebKitGTK now uses to expose the accessibility tree of the web content. While there are still improvements to be made, fixing this unblocks further developments.
+
+The existing accesibility support for GTK3 continues to work as before.
+
+### WebKit's DisplayLink support
+
+DisplayLink is a WebCore feature that improves resource utilization and improves synchronization with vertical screen retrace. 2.44 adds an implementation of this feature for the GTK port that improves rendering performance.
+
+### Improved hardware-acceleration video decoding and rendering
+
+If using GStreamer 1.24 or newer, the DMA-BUF sink supports DRM modifiers, which allow zero-copy negotiation with video decoders. This improves video decoding and rendering.
```

Comment:
```suggestion
Installations using GStreamer 1.24 or newer can take advantage of new support of DRM modifiers by the DMA-BUF sink. This improves video decoding and rendering because it allows zero-copy negotiation with the video decoders. 
```